Sources/swift-test-codecov/main.swift

```diff
@@ -123,11 +124,10 @@ struct StatsCommand: ParsableCommand {
             projectName: projectName
         )
 
-        if aggregateCoverage.totalCount == 0 {
+        if aggregateCoverage.totalCount == 0 && printFormat != .numeric {
             print("")
             print("No coverage was analyzed.")
             print("Double check that you are either running this tool from the root of your target project or else you've specified a project-name that has the exact name of the root folder of your target project -- otherwise, all files may be filtered out as belonging to other projects (dependencies).")
-            return
```

Comment:
Wanted to keep the helpful warning, but allow a minimum coverage fail and keep the numeric type only outputting the number to make piping the result easy.

Reply:
I think I like this change, but applicable to any output format.

What do you think about considering this a "warning" and creating a pair of flags for a boolean property that turns this warning off or on (on by default).

I'm tempted by the long-lived warning flag convention of `-Wxxxx` and `-Wno-xxxx` for turning the warning on and off, respectively. So `-Wmissing-tests` and `-Wno-missing-tests`. But that feels a bit off where there is already a convention used by this tool to have `--xxxx` and `--no-xxxx` argument pairs.

Ultimately, I think I like the following for the naming: `--warn-missing-tests` and `--no-warn-missing-tests`. You get the "no" naming for free since it matches precedent set by Swift Argument Parser.

Reply:
I was thinking something very similar. It's definitely a nice warning, but once you know it's working, not really necessary. 

I will add `--warn-missing-tests` and `--no-warn-missing-tests`.